halogen/index.d.ts

```diff
@@ -0,0 +1,85 @@
+// Type definitions for halogen
+// Project: https://github.com/yuanyan/halogen
+// Definitions by: Clément Devos <https://github.com/steller>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "halogen" {
+  import * as react from "react";
+
+  type VerticalAlign = "baseline" | "length" | "sub" | "super" | "top" | "text-top" | "middle" | "bottom" | "text-bottom" | "initial" | "inherit";
+
+  interface IHalogenCommonProps {
```

Comment:
I remove the `I` for the interface.
For the tslint I added the file, but it's don't nessessary. If you run `npm run lint -- halogen` it works without tslint.json. What it's a reason to added this file ?